docs/pages/versions/unversioned/sdk/clipboard.md

```diff
@@ -41,6 +90,6 @@ Sets the content of the user's clipboard.
 
 - **value (_string_)** -- The string to save to the clipboard.
 
-#### Returns
+## Returns
```

Comment:
```suggestion
#### Returns
```

Reply:
Also think we should change it here- https://github.com/expo/expo/pull/11882/files#diff-aa45b71f1c31fe9de7caa874725f7d65408980b81f3bc4f892e722989cb0a228L44

Reply:
Thanks, good eye, good catch :)